docs/content/en/docs/packages/emissary/addemissary.md

```diff
@@ -13,6 +13,7 @@ Be sure to refer to the [troubleshooting guide]({{< relref "../troubleshoot" >}}
   {{% alert title="Important" color="warning" %}}
    * Starting at `eksctl anywhere` version `v0.12.0`, packages on workload clusters are remotely managed by the management cluster.
    * While following this guide to install packages on a workload cluster, please make sure the `kubeconfig` is pointing to the management cluster that was used to create the workload cluster. The only exception is the `kubectl create namespace` command below, which should be run with `kubeconfig` pointing to the workload cluster.
+   * Operators can also leverage Ingress support provided by Cilium that uses the standard Kubernetes Ingress resource definition, with an `ingressClassName` of `cilium`. This can be used for path-based routing and for TLS termination. See [Kubernetes Ingress Support](https://docs.cilium.io/en/stable/network/servicemesh/ingress/) for details.
```

Comment:
@csplinter Do we want to embed Cilium commentary on packages documentation like this?